main.tf

```diff
@@ -143,9 +143,7 @@ resource "azurerm_kubernetes_cluster" "aks" {
     service_cidr       = "100.97.0.0/16"
   }
 
-  tags = {
-    Environment = "Development"
-  }
+  tags = merge({ Environment = "Development" }, var.aks_tags)
```

Comment:
I feel like this should be the default for the variable, not necessarily embedded in the module.

Reply:
Yeah, we can set it as the default.

The nice feature this allows is that the Environment is always set to Develop unless explicitly overridden in the aks_tags. It's nice to just add additional tags to the default values - it avoids having to repeat values when the default works well.